### PR TITLE
fix-9051: When selected an option on Text Rotate but it displayed wro…

### PR DESCRIPTION
**Pull request by lthanhhieu**
…ng on PDF file

Fixes #https://github.com/fossasia/open-event-frontend/issues/9051

#### Short description of what this resolves:
- When selected an option on Text Rotate but it displayed wrong on PDF file

#### Changes proposed in this pull request:
- When selected an option on Text Rotate but it displayed wrong on PDF file

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by khangon** [comment]
Hi @norbusan , @mariobehling , @cweitat ,
We have completed ticket https://github.com/fossasia/open-event-frontend/issues/9051
Please help me check and verify.
Thanks,

Screenshot from 2023-08-11 08-47-34

**Comment by mariobehling** [comment]
Thanks. Please resolve conflicts.

**Review by mariobehling — DISMISSED** [review]

**Comment by nnhathung** [comment]
Need to check after fixing conflict, please do not merge.

**Review by mariobehling — APPROVED** [review]

**Pull request merged**
